### ISSUE TITLE
Toggles for "Indicator for missing alt texts" and "Disable reminder to add alt text" keep resetting

### ISSUE THREAD

**Issue by anomopoda** [LucasGGamerM/moshidon#113]
**Describe the bug**

Same issue as https://github.com/LucasGGamerM/moshidon/issues/61#issue-1561224219 but with "Indicator for missing alt texts" and "Disable reminder to add alt text". Whenever I disable "Indicator for missing alt texts" and enable "Disable reminder to add alt text", they keep resetting to the default settings after app restart.

**To reproduce**

(copied from issue #61)

1. Go to 'Settings'
2. Click on the toggles for _"Indicator for missing alt texts"_ and _"Disable reminder to add alt text"_
3. Go to the home feed
4. Reboot the app
5. Both toggles have been reset to their default values

**Does this happen in the official app?**

The official Mastodon app does not have these options.

**Screenshots and screen recordings**

![image](https://user-images.githubusercontent.com/78587131/223734445-709ef2b1-4f35-4cb5-8d2a-5d7476a252a8.png)

![image](https://user-images.githubusercontent.com/78587131/223737360-ec2f013c-cc6d-4e05-86b4-982abf4c15c1.png)

**Version**

Moshidon version: v1.2.0+fork.99.moshinda
Android 11 (not rooted) and Android 13 (rooted)
